contracts/lib/MerklePatriciaProof.sol

```diff
@@ -129,7 +129,7 @@ library MerklePatriciaProof {
 
             if(currentNodeList.length == 17) {
                 if(pathPtr == path.length) {
-                    if(keccak256(abi.encodePacked(RLP.toBytes(currentNodeList[16]))) == value) {
+                    if(keccak256(abi.encodePacked(RLP.toData(currentNodeList[16]))) == value) {
```

Comment:
this PR, focused on tests, should *not* contain changes to the implementation

Reply:
Ok.